stdlib/array.gr

```diff
@@ -263,6 +264,8 @@ provide let concat = arrays => {
  * @param array: The array to copy
  * @returns The new array containing the elements from the input
  *
+ * @example Array.copy([> 1, 2, 3 ]) == [> 1, 2, 3 ]
```

Comment:
I think you should format the arrays and lists in these examples like how the formatter would